pkgs/by-name/wa/wakapi/package.nix

```diff
@@ -1,17 +1,23 @@
-{ lib, buildGoModule, fetchFromGitHub }:
-
-buildGoModule rec {
+{
+  lib,
+  buildGo123Module,
+  fetchFromGitHub,
+}:
+let
+  version = "2.12.0";
+in
```

Comment:
I get the idea is to not use `rec {` but will @r-ryantm be able to still do auto updates with that?

Reply:
yeah it can still update like this